package.json

```diff
@@ -29,6 +29,7 @@
     "postcss": "^8.1.0"
   },
   "dependencies": {
+    "@mdn/browser-compat-data": "^3.3.10",
```

Comment:
This dependency is 9 MB, 9 times bigger than `caniuse-lite`.

One of idea is to add selected `@mdn/browser-compat-data` data to `caniuse-lite`.

Reply:
Attempts to add the data to caniuse have been made - https://github.com/Fyrd/caniuse/pull/5722. They're denied because the data is already upstream in MDN. In cases like this idk what the best idea going forwards is?

Reply:
`caniuse-lite` is our own re-pack https://github.com/browserslist/caniuse-lite/

Can I Use provides `caniuse-db` package which is too big (12 MB vs 1 MB of our re-pack).

This is why it is easy to add MDN data to `caniuse-lite`. I do not like that `CANIUSE-lite` will have data not from Can I Use, but have no better idea.